mock/chairs/advice.md

```diff
@@ -0,0 +1,18 @@
+# Guidelines and Advice for Running a Group
+
+## Menu
+* [Advice](#advice)
+* [CSI](#csi)
```

Comment:
Suggest expanding the abbreviation from CSI to [whatever it stands for]

Reply:
Now added!